client/asset/eth/cmd/getgas/README.md

```diff
@@ -0,0 +1,39 @@
+## Gas Estimator Utility
+
+`getgas` is a program for getting gas estimates for DEX-critical operations
+on Ethereum. Use `getgas` to generate gas tables (`dexeth.Gases`) when adding a
+new token or a new contract version.
+
+If this is a new asset, you must populate either `dexeth.VersionedGases` or
+`dexeth.Tokens` with generous estimates before running `getgas`.
+
+### Use
+- Create a credentials file. `getgas` will look in `~/ethtest/getgas-credentials.json`. You can override that location with the `--creds` CLI argument. The credentials file should have the JSON format in the example below. The seed can be anything.
+
+**example credentials file**
+```
+{
+    "seed": "<32-bytes hex>",
+    "providers": {
+        "simnet": "http://localhost:38556",
+        "testnet": "https://goerli.infura.io/v3/<API key>",
+        "mainnet": "https://mainnet.infura.io/v3/<API key>"
+    }
+}
+```
+
+- Decide the maximum number of swaps you want in the largest initiation transaction, `--n`. Minimum is 2. `getgas` will check initiations with from 1 up to `n` swaps. There is a balance between cost and precision. Using more than 2 generates an average over `n - 1` intiations to calculate the cost of additional swaps (`Gases.SwapAdd`) and redeems (`Gases.RedeemAdd`).
+
+- If you run `getgas` with insufficient or zero ETH and/or token balance on the seed, no transactions will be sent and you'll get a message indicating the amount of funding needed to run.
```

Comment:
How do you the leftover funding out when done with the tool?

Reply:
I didn't think we would return funds. For ETH, I would just leave any remainder in the wallet for future estimation. For tokens, we're only funding a few atoms, almost definitely not worth the tx fees to send back.

Reply:
How do you deposit funds for a given seed?  You run it once and it fails with this zero balance error but it gives you an address?